### ISSUE TITLE
loss function 

### ISSUE THREAD

**Issue by whxueChris** [W-zx-Y/SM-PPM#3]
Thanks for sharing your great work! I have a  question about  the setting of loss function. You mentioned there was a balancing factor 0.5 before  ce loss, but i noticed the following code in `losses.py `. The loss function is only CE loss plus PCE loss.

`        probs = F.log_softmax(input, dim=1)

        probs = (probs * target_onehot)

        probs = torch.sum(probs, dim=1)

        probs = conf * probs + probs

        batch_loss = -probs

        if self.size_average:

            loss = batch_loss.mean()

        else:

            loss = batch_loss.sum()

        return loss`
